lib/puppet/module.rb

```diff
@@ -68,6 +68,17 @@ def initialize(name, path, environment)
     load_metadata
 
     @absolute_path_to_manifests = Puppet::FileSystem::PathPattern.absolute(manifests)
+
+    # i18n initialization for modules
+    if Puppet::GETTEXT_AVAILABLE
+      begin
+        initialize_i18n
```

Comment:
What I meant here @eputnam is that if `initialize_i18n` never `raises` an exception based on its code being inside a `begin` / `rescue` ... that means there's no need for a `rescue` here.

Merged for now, but this could probably be tightened up a little.